submission/loglist_manager.go

```diff
@@ -80,6 +98,7 @@ func (llm *LogListManager) RefreshLogList(ctx context.Context) (*LogListData, er
 	if err != nil {
 		return nil, err
 	}
+	logListLastRefresh.Set(float64(time.Now().Unix()))
```

Comment:
Does the gauge only support float64 values?  No int64 equivalent?

Reply:
I'm afraid interface expects float64